### PR TITLE
Blackjack game using codey coins

### LINKED ISSUE #105: Blackjack
Use library https://github.com/TimTheToaster/engine-blackjack-ts or build our own.

### PR DESCRIPTION
**Pull request by gzcharleszhang**
# Related Issues
resolves #105 

# Summary of Changes 
- Added a `.blackjack` or .`blj <amount>` command to start a blackjack game.
- Players can bet any amount between 10 and 1,000,000 inclusive.
- Players can react to H to hit, S to stand, or Q to quit.

# Steps to Reproduce
Start the bot and use `.blj`

# Screenshots
image
image

### CODE REVIEW

File: src/commands/games/blackjack.ts [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+      if (game.amountWon < game.bet) {
+        // player lost
+        return `You lost **${amountDiff}** Codey coin(s) ${getEmojiByName('codeySad')}, better luck next time!`;
+      }
+      if (game.amountWon > game.bet) {
+        // player won
+        return `You won **${amountDiff}** Codey coin(s) ${getEmojiByName('codeyLove')}, keep your win streak going!`;
+      }
+      // player tied with dealer
+      return `Tied! You didn't win or lose any Codey coin(s), try again!`;
```
Comment (Picowchew) [inline src/commands/games/blackjack.ts:163]: "or" -> "nor" probably.

**Review by Picowchew — APPROVED** [review]
In the "Summary of Changes", it looks like there is a typo for the maximum bet amount, since I believe that it is supposed to be 1 million.
Something to take note of is that while general members cannot remove the embed nor delete Codey bot's message, if someone with the required permissions accidentally removes the embed, then the game continues while the embed does not reappear, and more importantly, if the message is accidentally deleted, then the bot crashes.

File: src/commands/games/blackjack.ts (hunk 1 above)
Comment (gzcharleszhang) [reply src/commands/games/blackjack.ts:163]: oops i set it at 100k before but changed to 1m good catch

Comment (gzcharleszhang) [reply src/commands/games/blackjack.ts:163]: yeah I think that edge case is fine, I hope none of the admins remove those embeds, even if the bot crashes, the runtime will restart the bot immediately so not a big deal

**Comment by gzcharleszhang** [comment]
I will wait for one more person to review since this is a relatively big change

**Pull request merged**